test/inductor/test_torchinductor.py

```diff
@@ -1825,22 +1825,17 @@ def test_multilayer_var(self):
         def fn(a):
             return torch.var(a)
 
-        atol = None
-        rtol = None
-        if self.device == "cpu" and os.getenv("ATEN_CPU_CAPABILITY") == "default":
-            atol = 1e-4
-            rtol = 1e-4
         self.common(
             fn,
             ((torch.rand((10, 3, 352, 352), dtype=torch.float32),)),
-            atol=atol,
-            rtol=rtol,
+            atol=1e-3,
+            rtol=1e-3,
```

Comment:
This changes the tolerance for all devices. Is it expected?

Reply:
I just changed this back to how it was before in this PR, no new changes were introduced.